### PR TITLE
Added note about training mode in 'partial_fit'

### PR DESCRIPTION
**Pull request by chinmayapancholi13**
This PR adds a note about the mode used for training in the `partial_fit` function in the sklearn wrapper for LDA model.

### CODE REVIEW

File: gensim/sklearn_integration/sklearn_wrapper_gensim_ldamodel.py [hunk 1]
```diff
@@ -126,6 +126,8 @@ def get_topic_dist(self, bow, minimum_probability=None, minimum_phi_value=None,
     def partial_fit(self, X):
         """
         Train model over X.
+        By default, 'online' mode is used for training the LDA model.
+        Configure `passes` and `update_every` params at init to choose the mode among 'online', 'mini-batch', 'batch'
```
Comment (tmylk) [inline gensim/sklearn_integration/sklearn_wrapper_gensim_ldamodel.py]: Please add explicit values to use

Comment (chinmayapancholi13) [inline gensim/sklearn_integration/sklearn_wrapper_gensim_ldamodel.py]: I have updated the mode names and also added explicit values to use for each mode.

File: gensim/sklearn_integration/sklearn_wrapper_gensim_ldamodel.py [hunk 2]
```diff
@@ -126,6 +126,11 @@ def get_topic_dist(self, bow, minimum_probability=None, minimum_phi_value=None,
     def partial_fit(self, X):
         """
         Train model over X.
+        By default, 'online (single-pass)' mode is used for training the LDA model.
+        Configure `passes` and `update_every` params at init to choose the mode among :
```
Comment (menshikh-iv) [inline gensim/sklearn_integration/sklearn_wrapper_gensim_ldamodel.py]: Please remove spaces before `:` everywhere

Comment (chinmayapancholi13) [inline gensim/sklearn_integration/sklearn_wrapper_gensim_ldamodel.py]: Ok. Doing it right away.

**Pull request merged**